ping/ping.c

```diff
@@ -1372,7 +1377,7 @@ int ping4_receive_error_msg(struct ping_rts *rts, socket_st *sock)
 		if (rts->opt_flood) {
 			write_stdout("\bE", 2);
 		} else {
-			print_timestamp(rts);
+			print_timestamp("[", rts, "]");
```

Comment:
Does not this change output? Not aware of `[ ... ]`.

Reply:
OK, looking you moved `[` from `print_timestamp()`. Could you keep it there? Because than the formatting is duplicated on both places (ping IPv4 and IPv6), i.e. wrong direction.